src/main/java/seedu/address/logic/parser/PocketEstateParser.java

```diff
@@ -160,6 +161,16 @@ public Command parseCommand(String userInput) throws ParseException {
         case ListAppointmentCommand.COMMAND_WORD:
             return new ListAppointmentCommand();
 
+        case "add":
+        case "edit":
+        case "find":
+        case "sort":
+            throw new ParseException(Messages.missingPropertyAppointmentError(commandWord));
+
+        case "clear":
+        case "list":
+            throw new ParseException(Messages.missingAllPropertyAppointmentError(commandWord));
+
```

Comment:
I am fine with this but perhaps use static variables for this? Something like `INVALID_COMMAND_ADD`?

Reply:
its just a convenience function, i can use the static variable as well but it would look like "String.format(MESSAGE_MISSING_ALL_PROPERTY_APPOINTMENT, command, command, command,
                command, command, command)"

Reply:
Ohh i meant the "add", "find", "clear", etc

Reply:
So that they are not magic strings

Reply:
i see

Reply:
changed it!